tile-reduce/index.d.ts

```diff
@@ -0,0 +1,139 @@
+// Type definitions for tile-reduce 3.1.1
+// Project: https://github.com/mapbox/tile-reduce
+// Definitions by: Denis Carriere <https://github.com/DenisCarriere>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare namespace NodeJS  {
+  interface Global {
+    mapOptions: any
```

Comment:
If you want to declare a global variable, you can `declare global { declare var mapOptions: any; }`.